README.md

```diff
@@ -53,8 +54,20 @@ A customizable Neovim plugin to run code inside the editor
 
 ## Usage
 
+### Commands
+
+| Command           | Description                              |
```

Comment:
Add a table column to show which function each command runs, so it's more clear.